test/integration/index.js

```diff
@@ -4,18 +4,23 @@ const knex = require('../../knex');
 const logger = require('./logger');
 const config = require('../knexfile');
 const fs = require('fs');
+const path = require('path');
 
 const Bluebird = require('bluebird');
 
 Object.keys(config).forEach((dialectName) => {
   return require('./suite')(logger(knex(config[dialectName])));
 });
 
-after(function(done) {
+before(function(done) {
   if (config.sqlite3 && config.sqlite3.connection.filename !== ':memory:') {
-    fs.unlink(config.sqlite3.connection.filename, function() {
-      done();
-    });
+    fs.copyFile(
+      path.resolve(__dirname, '../sample.sqlite3'),
```

Comment:
Considering that this is a very specific db sample, can name be made more clear? E. g. "multilineCreateMaster.sqlite3"

Reply:
Done.